src/shared/auth-context.ts

```diff
@@ -15,5 +16,6 @@ export const AuthContext = createContext<User>({
     displayName: '',
     photoURL: '',
     expectedDueDate: null,
-    setExpectedDueDate: () => {}
+    setExpectedDueDate: () => {},
+    setDisplayNameOnCreate: () => {}
 })
```

Comment:
Unsure if including lots of methods to the context is best practice or not but this solves the problem at hand for now